Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Net.NetworkInformation;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            for (int i = 0; i < 2; i++)
+            {
+                Console.WriteLine("What is your first name?");
+                var name = Console.ReadLine();
+
+                Console.WriteLine("What is your last name?");
+                var surname = Console.ReadLine();
+
+                Console.WriteLine("What is your age?");
+                var age = int.Parse(Console.ReadLine());
+
+                Console.WriteLine("What is your weight (in lbs)?"); //Murica
+                var weight = int.Parse(Console.ReadLine());
+
+                Console.WriteLine("What is your height (in inches)?"); //Murica
+                var height = Single.Parse(Console.ReadLine());
+
+                Console.WriteLine(name + " " + surname + " is " + age + " years old, his weight is " + weight +
+                    " lbs and his height is " + height + " in.");
+
+                const int dumbAmericanBMIConstant = 703;
+                float BMI = (weight / (height * height)) * dumbAmericanBMIConstant;
```

Comment:
Conventionally, it should be `bmi` nor BMI. If it was a method name, it would be `Bmi`.